build.gradle.kts

```diff
@@ -8,12 +7,13 @@ plugins {
     id("com.github.johnrengelman.shadow") version "7.0.0"
 }
 
-configure<JavaPluginConvention> {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = sourceCompatibility
+plugins.withId("java") {
+    the<JavaPluginExtension>().toolchain {
+        languageVersion.set(JavaLanguageVersion.of(16))
+    }
 }
 
-version = "5.1.1"
+version = "6.0.0"
```

Comment:
If we're doing a major version bump anyway, we can rebrand the classpath from com.empcraft to com.plotsquared like our other addons do.